ote/src/cljc/ote/time.cljc

```diff
@@ -30,6 +33,14 @@
          t/to-default-time-zone
          (format/unparse (format/formatter "dd.MM.yyyy HH:mm"))))))
 
+#?(:cljs
+   (defn date-fields-from-timestamp [timestamp]
+     (if  (nil? timestamp)
+       " " ;: if nil - print empty string
```

Comment:
This function returns string (" ") or a date fields map?
Either move `nil` check to call site or return `nil` here as well.